src/petals/server/handler.py

```diff
@@ -323,9 +331,9 @@ def _serialize_grads(
     ) -> Sequence[runtime_pb2.Tensor]:
         """Serialize backward gradients w.r.t. inputs using either default schema or custom user-specified schema"""
         # Modify grad_inputs_schema to support grad_prompts
-        assert len(requested_backends[0].args_schema) == 1 and len(grads) in (1, 2)  # TODO generalize
+        assert len(requested_backends[0].args_schema) == 2 and len(grads) in (1, 2)  # TODO generalize
```

Comment:
From conversation with @borzunov vs @artek0chumak vs @justheuristic 

We need a way to support (temporarily) a case where old client sends a request to the new server and vice versa
One way we could do that is to add code to `rpc_*` methods that checks for the right number of inputs and, if missing, adds an empty attention mask